Strings/Skills/Murasama's Pestilence.md

```diff
@@ -0,0 +1,15 @@
+# Murasama's Pestilence
+
+| Language | Name |
+| :------- | :---------- |
+| Text ID | 9063 |
+| Note | active but not in inventory<br>may get reworked in future |
+| | |
```

Comment:
This spacing seems okay but would be good to maintain this everywhere. I dont see the same in [here](https://github.com/SekiroResurrection/mod-files/pull/2/files?short_path=cfb2e9d#diff-cfb2e9d53681ef69d6cf0b788a6af4d208779530c6e0ca024b63b3ddedaff23e)

@PrattiDev Whats your opinion?

Reply:
To clarify by spacing I mean the extra rows between each row

Reply:
I'd like to recommend using a spacing row only for Skills, Items, and Weapons which have several sentences per translations.
NPCs and EventTexts have one sentences each within several tables, so adding spacing between rows may reduces readability.